cypress/e2e/accessibility.cy.ts

```diff
@@ -0,0 +1,30 @@
+describe("Accessibility tests", () => {
```

Comment:
looks like the rebase didn't really work out - these changes shouldn't be on your branch!
they should be on main (which would be 'beneath' your branch), so the changes would be present, but not as an addition or as separate commits...
May need an interactive rebase (to squash some commits or delete some commits) to fix this up!

Reply:
resolved as suggested